packages/client/src/index.ts

```diff
@@ -5,6 +5,8 @@ export * from "./getChainId";
 export * from "./getEndpoint";
 export * from "./getFees";
 export * from "./getForeignCoins";
+export * from "./getHardhatConfig";
+export * from "./getHardhatConfig";
```

Comment:
It looks like lines 8 and 9 duplicate each other.

Reply:
59aa44c